cli.js

```diff
@@ -15,6 +17,7 @@ const cli = meow(`
 `);
 
 const input = cli.input[0];
+const bytesFlag = cli.flags.b;
```

Comment:
I would like a long flag too: `--pretty`. You can use the minimist `alias` option to create the short flag for it.